src/utils/index.js

```diff
@@ -57,6 +69,52 @@ export const buildSchema = (customNodes = {}, customMarks = {}) => {
 	});
 };
 
+const renderReactFromSpec = (elem, key, holeContent) => {
```

Comment:
Can you leave a little note about what's happening in this function? Without squinting at it too closely, it looks like a homegrown `ReactDOM.render` implementation. Referencing `elem.$$typeof` on a React element seems like a bit of a yellow flag in userspace code.

Reply:
Definitely, added in 73ff7b3.